transport/http/handler_test.go

```diff
@@ -135,7 +135,7 @@ func TestHandlerInternalFailure(t *testing.T) {
 
 	rpcHandler := transporttest.NewMockHandler(mockCtrl)
 	rpcHandler.EXPECT().Handle(
-		gomock.Any(),
+		transporttest.NewContextMatcher(t, time.Second),
```

Comment:
What are you testing here? Just passing in a `time.Second`? I would expect some interaction with TTL here